Card/src/main/java/com/paypal/android/card/Card.kt

```diff
@@ -23,5 +25,20 @@ data class Card(
     /**
      * The card's security code (CVV, CVC, CVN, CVE, or CID)
      */
-    var securityCode: String = ""
+    var securityCode: String? = null,
+
+    /**
+     * The card holder's name as it appears on the card
+     */
+    var name: String? = null,
+
+    /**
+     * The portable international postal address
+     */
+    var billingAddress: Address? = null,
+
+    /**
+     * Additional attributes associated with the use of this card
+     */
+    var attributes: CardAttributes? = null
```

Comment:
It feels a little arbitrary which params are in `CardAttributes` vs in `Card`

Reply:
another note is that `CardAttributes` is marked `LIMITED_RELEASED`
If it's like how `validate-payment-method` being limited vs `confirm-payment-source` being publicly exposed, I wonder maybe `CardAttributes` was also created only for some internal use case, and maybe we don't need to expose this to merchants? Maybe we should confirm that :smile: